examples/basic/default/Makefile

```diff
@@ -27,7 +27,10 @@ DEVELHELP ?= 1
 # Change this to 0 show compiler invocation lines by default:
 QUIET ?= 1
 
+CFLAGS += "-DCONFIG_NANOCOAP_BLOCK_SIZE_EXP_MAX=10"
```

Comment:
Does this need to be this block size? If yes, please make it configurable via Kconfig (`app.config`) so users do not run into the `CFLAGS` being defined twice.